### ISSUE TITLE
help: scoped composable keeps overriding itself

### ISSUE THREAD

**Issue by RobinBertilsson** [nuxt-community/composition-api#266]
I'm not sure if this is a bug or me being stupid, but I have a composable, responsible for making API calls using the `useAsync` function, this composable is used inside my `pages/index.vue` but also inside `layouts/default.vue`.

At first, it seems like everything is working as expected, but after about 1 second the content which `pages/index.vue` should receive ends up inside `layouts/default.vue`?

We have a big release on Monday, and this is killing me... HELP ❤️

**composable/use-cms.vue**
```
function useCms($axios: NuxtAxiosInstance, path: string) {
  const { error } = useContext()

  const content = useAsync(async () => {
     try {
       return (await $axios.get(`https://api.cms.com/slug/${path}`)).data.story.content
     } catch (e) {
       error(e)
     }
  })

  return {
    content,
  }
}

export default useCms
```

**pages/index.vue**
```
<template>
  <div>
    <div>{{ content }}</div>
  </div>
</template>

<script lang="ts">
import { defineComponent } from 'nuxt-composition-api'
import useCms from '~/composables/use-cms'

export default defineComponent({
  setup(props, { root: { $axios }}) {
    const { content } = useCms($axios, 'index')

    return {
      content,
    }
  }
})
</script>
```

**layouts/default.vue**
```
<template>
  <div>
    <div>{{ content }}</div>
    
    <main>
      <nuxt />
    </main>
  </div>
</template>

<script lang="ts">
import { defineComponent } from 'nuxt-composition-api'
import useCms from '~/composables/use-cms'

export default defineComponent({
  setup(props, { root: { $axios }}) {
    const { content } = useCms($axios, 'settings')

    return {
      content,
    }
  }
})
</script>
```

### Expected output (HTML)
```
<div>
  <div>content from layouts/default.vue</div>
  
  <main>
    <div>
      <div>content from pages/index.vue</div>
    </div>
  </main>
</div>
```

### Actual output
```
<div>
  <div>content from pages/index.vue</div> <- THIS IS THE WIERD PART, IT'S THE CONTENT FROM DOWN BELOW
  
  <main>
    <div>
      <div>content from pages/index.vue</div>
    </div>
  </main>
</div>
```

**ℹ️ Additional context**
- I'm using Universal / SSR mode.
- nuxt-composition-api@0.9.3

**Comment by RobinBertilsson** [comment]
@danielroe, what a hero, it worked! ⭐️

Do you mind explain what's going on here?
Also, I think this might be a common issue, so explaining it shortly on the following page might be a smart idea?
https://composition-api.nuxtjs.org/useAsync

What do you think?

Again, thanks a lot!

**Comment by danielroe** [comment]
@RobinBertilsson It's in the [`ssrRef` section of the docs](https://composition-api.nuxtjs.org/ssrRef) - but it might be a good idea to add a note in `useAsync` too.

**Comment by danielroe** [comment]
closed in a072c94148f1516aa144c6744ae1787e6485974b
